app/controllers/articles_controller.rb

```diff
@@ -13,17 +13,25 @@ def new
 
   def send_last_10
     @articles = Article.last(10)
-    UserMailer.last_10_email(current_user,@articles).deliver
+    UserMailer.last_10_email(current_user.email, @articles).deliver_now
```

Comment:
Now that you are using sidekiq. It would be nice to send all the emails through it. Be carefull that if you use sidekiq you can't send @articles. You should send the ids